### PR TITLE
(docs): add alert boxes

### LINKED ISSUE #6957: [DOCS] Alert Boxes
### Description

We should find a plugin (or build one) that allows us to display alert boxes in the documentation - specifically for calling out important information that is commonly missed.

### Screenshots

Here's what an alert box looks like in Becca's documentation:

image

### Additional information

This doesn't stand out enough and is often missed by contributors:

image

### PR DESCRIPTION
**Pull request by NiazMorshed2007**
Closes #6957 

Wrapping the important message with Alert component in mdx file produces expected style discussed in the issue.
Currently, I'm using `type` of `warning` in `Alert` component. (different views for different types are discussed in #6957 )

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7074"></a>

### CODE REVIEW

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution 👍 

- please add a screenshot in the PR description
- GitHub Action is failing `./pages/docs/quickstart.mdx
Module not found: Can't resolve '@components/alert.js'` I think this needs to be `@components/Alert.js` as it is case sensitive

**Comment by NiazMorshed2007** [comment]
@eddiejaoude fixed!
And here is a screen shot of it currently looks like

image

If you want to see the different views for different types please check my message on #6957

**Comment by eddiejaoude** [comment]
Thank you 👍 The markdown does not render, I think we have 2 options

- add functionality for the markdown, this can come later
- remove markdown from the messages

**Comment by eddiejaoude** [comment]
Also I think this PR should update all "important" information in the docs, for example https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/docs/environments/local-development.mdx?plain=1#L6

**Comment by NiazMorshed2007** [comment]
@eddiejaoude All good. I'm currently removing markdown from the message.
And updating all the important messages ✌️

**Comment by eddiejaoude** [comment]
Great! thank you 👍

**Comment by eddiejaoude** [comment]
@NiazMorshed2007 do you need any help?

**Comment by NiazMorshed2007** [comment]
Hey @eddiejaoude ,
Sorry for the late. I was going busy for the Appwrite hackathon,, 
Check this commit & let me know if any changes are needed.

Note: I've updated the `message` props with `jsx` instead of `string` to add bold/italic/other (because we can't use markdown) formats in message.

**Comment by NiazMorshed2007** [comment]
Conflict resolved.
But, does the warning message in the gitpod docs looks fine?

image

**Comment by eddiejaoude** [comment]
Yes I think it looks good 👍 

Are there any more alerts to be added to the docs?

**Comment by NiazMorshed2007** [comment]
All good.
I checked all docs, all covered!

File: pages/docs/quickstart.mdx [hunk 1]
```diff
@@ -114,14 +115,31 @@ To add more links, add another object inside the links collection. For example:
...
 You can skip `optional extended message`
 
-**NOTE**: Please use the online [JSON Lint](https://jsonlint.com) tool to check if your JSON file is _error free_ before committing your pull request.
+<Alert
+  type="warning"
+  message={
+    <>
+      <strong>NOTE</strong>: Please use the online{" "}
+      <Link href="https://jsonlint.com">JSON Lint</Link> tool to check if your
+      JSON file is <i>error free</i> before committing your pull request.
```
Comment (eddiejaoude) [inline pages/docs/quickstart.mdx:124]:
```suggestion
      <strong>NOTE</strong>: Please use our online{" "}
      <Link href="https://linkfree.io/playground">Playground</Link> tool to check your
      JSON file is <I>valid</i> before committing your pull request.
```

File: pages/docs/quickstart.mdx [hunk 2]
```diff
@@ -114,14 +115,31 @@ To add more links, add another object inside the links collection. For example:
...
 You can skip `optional extended message`
 
-**NOTE**: Please use the online [JSON Lint](https://jsonlint.com) tool to check if your JSON file is _error free_ before committing your pull request.
+<Alert
+  type="warning"
+  message={
+    <>
+      <strong>NOTE</strong>: Please use our online{" "}
+      <Link href="https://linkfree.io/playground">Playground</Link> tool to check your
+      JSON file is <I>valid</i> before committing your pull request.
```
Comment (eddiejaoude) [inline pages/docs/quickstart.mdx:124]:
```suggestion
      JSON file is <i>valid</i> before committing your pull request.
```

File: pages/docs/environments/gitpod.mdx [hunk 3]
```diff
@@ -17,11 +19,32 @@ There are 2 ways you can open a repository with Gitpod:
...
-1. Be sure that you have a fork of LinkFree in your Github Profile. If you don't have it, check out the [Quickstart Guide](https://linkfree.io/docs/quickstart).
-
-2. When you have your environment running the database will be empty. You need to visit the path `/api/system/reload?secret=development` to load the json files.
+<Alert
+  type="warning"
+  message={
+    <>
+      <i>
+        <strong>NOTE:</strong>
+      </i>{" "}
```
Comment (eddiejaoude) [inline pages/docs/environments/gitpod.mdx:28]: This note seems to have italic but the other note does not.

**Review by eddiejaoude — APPROVED** [review]
Looks good! Thank you 👍 

 I will merge and fix the inline comment in a temporary branch

**Pull request merged**